media-libs/libshout/libshout-2.4.1.recipe

```diff
@@ -49,8 +66,13 @@ BUILD_PREREQUIRES="
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
+defineDebugInfoPackage libshout$secondaryArchSuffix \
+	$libDir/libshout.so.3.2.0
+
 BUILD()
 {
+	export LIBS="-lnetwork"
+	echo "AC_CONFIG_MACRO_DIRS([m4])" >> configure.ac
```

Comment:
For each build, this would append this line.
Maybe move this to PATCH().

Reply:
Will check later, thanks! :+1:

Reply:
Moved it to PATCH()